Library/Homebrew/cleanup.rb

```diff
@@ -261,6 +264,8 @@ def clean!(quiet: false, periodic: false)
           end
           cleanup_cask(cask) if cask
         end
+
+        Cleanup.autoremove(dry_run: dry_run?) if Homebrew::EnvConfig.autoremove?
```

Comment:
```suggestion
```

I think we can skip this; it's only invoked if you do `brew cleanup wget` or similar and I don't think we want autoremove then.

Reply:
Ah, okay. I was wondering about that one.